File: Problem Solving/Algorithms/Strings/Pangrams/Pangrams.cpp
#include <bits/stdc++.h>

using namespace std;

// Complete the pangrams function below.
string pangrams(string s) {
    set<char> st;
    for (char ch : s)
        if (ch != ' ') st.insert(tolower(ch));
    if (st.size() == 26) return "pangram";
    else return "not pangram";


}
int main()
{
    ofstream fout(getenv("OUTPUT_PATH"));

    string s;
    getline(cin, s);

    string result = pangrams(s);

    fout << result << "\n";

    fout.close();

    return 0;
}

File: Problem Solving/Algorithms/Strings/Pangrams/Pangrams.py
#!/bin/python3

import math
import os
import random
import re
import sys

# Complete the pangrams function below.
def pangrams(s):
    return ['not pangram', 'pangram'][len({*s.lower()}) == 27]

if __name__ == '__main__':
    fptr = open(os.environ['OUTPUT_PATH'], 'w')

    s = input()

    result = pangrams(s)

    fptr.write(result + '\n')

    fptr.close()
